### ISSUE TITLE
No event is fired when changing values of boolean attributes by clicking on toggles

### ISSUE THREAD

**Issue by berry4u** [josdejong/jsoneditor#1203]
Can someone suggest me a workaround?

**Comment by josdejong** [comment]
I just tried change a boolean by clicking the toggle, and it fires `onChange`, `onChangeJSON`, and `onChangeText`. How can I reproduce your issue?

**Comment by berry4u** [comment]
Yes what you said is right, what I meant was to consider to fire even the onEvent  which is more useful in order to quickly find what attribute has been clicked because onEvent returns path, while other events return the whole string or json.

**Comment by josdejong** [comment]
Ah, I understand what you mean.

Are you using the latest version of jsoneditor? This issue was addressed not long ago in `v9.1.2`, see #1126
